pnpm-lock.yaml

```diff
@@ -243,6 +243,7 @@ importers:
       remark-smartypants: ^2.0.0
       remark-toc: ^8.0.1
       tailwindcss: ^3.1.8
+      typed.js: ^2.0.12
```

Comment:
Hmmm, i get what @olayway is saying. We really don't want to have to add this to core to have this for the main site. I guess we are getting to point where we want website to be its own app properly. This is something worth flagging IMO. For now we live with it but we definitely want to flag for ourselves as we move forward.